app/setup.py

```diff
@@ -288,6 +278,25 @@ def setup_datastore(application):
     application.eq['storage'] = DatastoreStorage(client)
 
 
+def setup_submitter(application):
+    if application.config['EQ_SUBMITTER'] == 'gcs':
+        application.eq['submitter'] = GCSSubmitter(
+            project_id=application.config['EQ_GCS_SUBMITTER_PROJECT_ID'],
+            bucket_name=application.config['EQ_GCS_SUBMITTER_BUCKET_ID'],
+        )
+    elif application.config['EQ_SUBMITTER'] == 'rabbitmq':
```

Comment:
Since we have removed `get_env_or_fail` from the `settings.py` we should add a check here to kill the app if required values are missing.
Currently the app fails to start if required values are missing causing deployment to fail and app to not be deployed.
With this current setup, the deployment will succeed with an app that will not be able to submit.

Reply:
Have addressed this but we do have cross dependencies, so if `EQ_SUBMITTER` is 'gcs' but then the PROJECT ID or BUCKET_ID might not be set. Ideally, we should enforce groups of dependent env variables. When speaking to Mark, he said we can address that in a follow up card.

Reply:
Added code to check the dependant env variable in this case.